src/components/forms/CreateUnitLabelsForm.js

```diff
@@ -52,6 +53,19 @@ const CreateUnitLabelsForm = ({ value, onChange }) => {
     [selectedWayToAddLabel],
   );
 
+  useEffect(() => {
+    if (
+      !_.isEmpty(_.filter(Object.values(value), value => value)) &&
+      !selectedWayToAddLabel
+    ) {
+      if (!value.id && JSON.parse(localStorage.getItem('selectedLabel'))) {
```

Comment:
would this crash if "selectedLabel" is not in localstorage?

Reply:
The local storage only adds the selection of "Add New Label" (Which I now realize I should change the name to instead of "selectedLabel") inside when it is selected.  If there is no localStorage of it, the if statement will return false because localStorage will return null.

Reply:
yes but your trying to parse the localstorage response

JSON.parse(NOT JSON) will crash right?

Reply:
Oh I see what you mean.  No, It won't crash because since I have to add an object to localStorage I have to stringify it first when I input it inside the localStorage and in order to get the response back correctly I have to parse It. I also tested it a few times to make sure and I haven't had any issues

I stringify It here
<img width="348" alt="Screen Shot 2022-05-05 at 8 39 58 AM" src="https://user-images.githubusercontent.com/14043581/166960661-d348dec8-bec0-41c2-b2cb-cd3328f7c86f.png">
nside the localStorage and in order to get the response back I have to parse It

Reply:
yea but what happens if someone clears localStorage and this doesnt get repopulated before this runs?

Reply:
I realized I didn't need the localStorage logic and removed it. I replaced it with a more logical simple code